enterprise/frontend/src/metabase-enterprise/content_translation/use-translate-content.unit.spec.tsx

```diff
@@ -0,0 +1,148 @@
+import { setupEnterprisePlugins } from "__support__/enterprise";
+import { mockSettings } from "__support__/settings";
+import { renderHookWithProviders } from "__support__/ui";
+import { useListContentTranslationsQuery } from "metabase-enterprise/api";
+import type { TokenFeatures } from "metabase-types/api";
+import type { RetrievedDictionaryArrayRow } from "metabase-types/api/content-translation";
+import {
+  createMockTokenFeatures,
+  createMockUser,
+} from "metabase-types/api/mocks";
+import { createMockState } from "metabase-types/store/mocks";
+
+import { useTranslateContent } from "./use-translate-content";
+
+jest.mock("metabase-enterprise/api", () => ({
+  useListContentTranslationsQuery: jest.fn(),
+}));
+
+const mockUseListContentTranslationsQuery =
+  useListContentTranslationsQuery as jest.Mock;
+
+interface SetupOpts {
+  locale: string | undefined;
+  /** An untranslated string */
+  msgid: string | null | undefined;
+  translations: RetrievedDictionaryArrayRow[] | undefined;
+  tokenFeatures?: Partial<TokenFeatures>;
+  hasEnterprisePlugins?: boolean;
```

Comment:
again, since you're testing this hook directly, bypassing all the enterprise conditionals, you shouldn't need token features or enterprise plugins args.